simphony_ui/liggghts_wrapper_creation.py

```diff
@@ -69,3 +69,54 @@ def create_liggghts_wrapper(liggghts_settings):
     ]
 
     return liggghts_wrapper
+
+
+def create_liggghts_datasets(liggghts_wrapper, liggghts_settings):
+    """ Creates the liggghts particles datasets from the settings as provided
+    by the model object
+
+    Parameters
+    ----------
+    liggghts_wrapper : LiggghtsWrapper
+        Your liggghts wrapper in which you want to put the datasets
+    liggghts_settings : LiggghtsModel
+        The traited model describing the liggghts parameters
+
+    Returns
+    -------
+    flow_dataset :
+        A dataset containing flow particles
+    wall_dataset :
+        A dataset containing wall particles
+    """
+    input_file = liggghts_settings.input_file
+
+    list_particles = liggghts.read_data_file(input_file)
```

Comment:
Ok, here I need to talk with you if this is a problem with consistency. We don't want one routine to get the filename, the other the file content. I am referring to the other PR, for the mesh build. I am not 100 % sure though. We'll take a look at it tomorrow.